pkg/taskutil/taskutil.go

```diff
@@ -117,6 +117,8 @@ func NewTask(ctx context.Context, client *containerd.Client, container container
 			return nil, err
 		}
 		ioCreator = cio.LogURI(u)
+	} else if flagD && logURI == "" {
```

Comment:
Could you please add comment for this specific case!

Reply:
Ping @Howie59